src/webhook/mutation/pod_mutator.go

```diff
@@ -725,18 +733,16 @@ func updateContainerOneAgent(c *corev1.Container, dk *dynatracev1beta1.DynaKube,
 		})
 
 	if dk.Spec.Proxy != nil && (dk.Spec.Proxy.Value != "" || dk.Spec.Proxy.ValueFrom != "") {
-		c.Env = append(c.Env,
-			corev1.EnvVar{
-				Name: "DT_PROXY",
-				ValueFrom: &corev1.EnvVarSource{
-					SecretKeyRef: &corev1.SecretKeySelector{
-						LocalObjectReference: corev1.LocalObjectReference{
-							Name: dtwebhook.SecretConfigName,
-						},
-						Key: "proxy",
-					},
-				},
-			})
+		secretConfig, err := m.getSecretConfigData(nsName)
+		if err != nil {
+			podLog.Error(err, "Failed to inject proxy info into pod", "name", pod.Name, "generatedName", pod.GenerateName)
```

Comment:
```suggestion
			podLog.Error(err, "failed to inject proxy info into pod", "name", pod.Name, "generatedName", pod.GenerateName)
```

Reply:
Fixed init capped strings in all `podLog.Error` calls.